Fixtures/ConsumerManagerStub.php

```diff
@@ -0,0 +1,65 @@
+<?php
+declare(strict_types = 1);
+
+namespace M6Web\Bundle\KafkaBundle\Fixtures;
+
+use M6Web\Bundle\KafkaBundle\AbstractManager;
+use M6Web\Bundle\KafkaBundle\Consumer\ConsumerManager;
+use M6Web\Bundle\KafkaBundle\Consumer\TopicsConsumptionState;
+use M6Web\Bundle\KafkaBundle\Tests\Units\Producer\ProducerManager;
+
+/**
+ * Class ConsumerManagerStub
+ * @package M6Web\Bundle\KafkaBundle\Fixtures
+ *
+ * A stub class to imitate ConsumerManager class
+ */
+class ConsumerManagerStub extends ConsumerManager
+{
+    /**
+     * @param \RdKafka $entity
+     * @return AbstractManager
+     */
+    public function setEntity(\RdKafka $entity): AbstractManager
+    {
+        return $this;
+    }
+
+    /**
+     * @param string $brokers
+     * @return AbstractManager
+     */
+    public function addBrokers(string $brokers): AbstractManager
+    {
+        return $this;
+    }
+
+    /**
+     * @param int $logLevel
+     * @return AbstractManager
+     */
+    public function setLogLevel(int $logLevel): AbstractManager
+    {
+        return $this;
+    }
+
+    /**
+     * @param string             $name
+     * @param \RdKafka\TopicConf $rdKafkaTopicConf
+     * @param array|null         $confToSet
+     * @return $this
```

Comment:
pas d'accord, `@return $this` quand on retourne `$this` c'est une bonne pratique IMHO : 
- c'est plus précis que `@return self` parce que ça indique que c'est vraiment la même instance qui est retournée
- c'est supporté par PHPStorm
- c'est dans le draft de la PSR sur la php doc